assembl/static2/js/app/components/debate/survey/question.jsx

```diff
@@ -78,19 +79,21 @@ class Question extends React.Component {
     const maxChars = this.txtarea.props.maxLength;
     const { questionId, scrollToQuestion, index, refetchTheme } = this.props;
     const body = this.state.postBody;
-    this.props.mutate({ variables: { ideaId: questionId, body: body } })
-    .then(() => {
-      scrollToQuestion(true, index + 1);
-      displayAlert('success', I18n.t('debate.survey.postSuccess'));
-      refetchTheme();
-      this.setState({
-        postBody: '',
-        showSubmitButton: false,
-        remainingChars: maxChars
+    this.props
+      .mutate({ variables: { ideaId: questionId, body: body } })
+      .then(() => {
+        scrollToQuestion(true, index + 1);
+        displayAlert('success', I18n.t('debate.survey.postSuccess'));
+        refetchTheme();
```

Comment:
From server or from Apollo cache? Can't see that in this PR. Its likely from PR #103